openapi/src/main/openapi/teams/path.yaml

```diff
@@ -1,5 +1,6 @@
 teams:
   get:
+    deprecated: true
```

Comment:
blocking: 
Shouldn't we:
 * have a since?
 * explain what to do instead?

My points:
 * they will be able to compute when it can be deleted
 * it ease the transition